### PR TITLE
emulator bump to v1.1.20

### PR DESCRIPTION
**Pull request by hlshen**

### CODE REVIEW

**Review by fredzqm — APPROVED** [review]

**Review by fredzqm — COMMENTED** [review]
Include change logs

```
[added] Added Postgres 16 support.
```

**Comment by fredzqm** [comment]
https://github.com/firebase/firebase-tools/blob/63fdf2c6cf8d9da0e4080e85892421f3eeab6e9e/src/emulator/dataconnectEmulator.ts#L62 this will be broken

**Review by fredzqm — CHANGES_REQUESTED** [review]
https://github.com/firebase/firebase-tools/blob/63fdf2c6cf8d9da0e4080e85892421f3eeab6e9e/src/emulator/dataconnectEmulator.ts#L62

This will be broken. 
Can you work with Rosalyn to break it into a separate call?
```
curl <localAddress>/emulator/configure -X POST -d '{
  "connection_string": "...",
}'
```

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,3 +2,4 @@
 - Updated Pub/Sub emulator to v0.8.14 (#7197).
 - Updates default resource names and template during `firebase init dataconnect`.
 - Improved handling of 'fresh from Console' services during `init dataconnect`.
+- Update to Firebase Data Connect Emulator v1.1.20 which add support for Postgres 16
```
Comment (rosalyntan) [inline CHANGELOG.md:5]: nit: adds

**Review by rosalyntan — APPROVED** [review]

**Pull request merged**